pkg/importer/openapi.go

```diff
@@ -337,10 +337,14 @@ func (l *loader) buildParams(params openapi3.Parameters) Parameters {
 }
 
 func (l *loader) buildParam(p *openapi3.Parameter) Param {
+	name := p.Name
+	if hasToBeGoSafe(p.In) {
```

Comment:
this actually needs to be sysl safe, not go safe. the - breaks the sysl parser

Reply:
changed to sysl safe